app/components/login.tsx

```diff
@@ -0,0 +1,117 @@
+'use client'
+import Link from 'next/link'
+import { useState, FormEvent } from 'react'
+import Image from 'next/image'
+import profile from './profile.svg'
```

Comment:
@mohd-khan09 You can move this image to separate `assets` folder and update the path